app/api/notifications.py

```
@@ -0,0 +1,517 @@
+    
+    def disconnect(self, user_id: uuid.UUID):
+        """Disconnect a user's WebSocket."""
+        if user_id in self.active_connections:
+            del self.active_connections[user_id]
+    
+    async def send_personal_message(self, message: dict, user_id: uuid.UUID):
+        """Send message to specific user."""
+        if user_id in self.active_connections:
+            try:
+                await self.active_connections[user_id].send_json(message)
+            except:
```

Ruff (0.13.2)

Do not use bare `except`

(E722)